src/components/questionsSetScreen/questionsSetScreen.component.js

```diff
@@ -18,16 +18,16 @@ export const QuestionsSetScreenComponent = ({
       >
         { isChooseModeDialog ? renderChooseMode() : null }
         <ImageBackground
-          source={img}
+          source={{ uri: img }}
           style={styles.header}
           imageStyle={styles.borderRadius}
         >
           <Text style={styles.headerText}>{headerTitle}</Text>
         </ImageBackground>
         <FlatList
           style={styles.item}
-          data={titles}
-          renderItem={(item) => renderItem(item)}
+          data={categoryTopics}
+          renderItem={(categoryTopic) => renderItem(categoryTopic)}
```

Comment:
renderItem={(categoryTopic) => renderItem(categoryTopic)} ->  renderItem={renderItem}
 do you know why you need to create a separate function for renderItem prop?